manifests/b/Baidu/SceneInput/2.0.0.1043/Baidu.SceneInput.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS1.CRLF.7-3-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Baidu.SceneInput
+PackageVersion: 2.0.0.1043
+PackageLocale: en-US
+Publisher: 百度在线网络技术（北京）有限公司
+PublisherUrl: https://www.baidu.com/
+PublisherSupportUrl: https://tieba.baidu.com/f?kw=%E7%99%BE%E5%BA%A6%E8%BE%93%E5%85%A5%E6%B3%95
+PrivacyUrl: http://privacy.baidu.com/policy
+Author: Baidu Online Network Technology (Beijing) Co., Ltd.
+PackageName: 袋鼠输入
+# PackageUrl: https://daishu.baidu.com/
```

Comment:
Unfortunately, the PackageUrl presenting an HTTP response `502 Bad Gateway` prevents navigating from it to the InstallerUrl. Does this package have any other PackageUrl?

Reply:
No, it has been broken for about a year. Maybe you can see it through Internet Archive.